### PR TITLE
chore: rename homeview section names

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [ONYX-1242] 

### Description

Rename HomeView section names

**This PR is blocked by** https://github.com/artsy/metaphysics/pull/5962

https://github.com/user-attachments/assets/0c995fa6-441b-4676-a892-7ea62b21d3d2

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-1242]: https://artsyproduct.atlassian.net/browse/ONYX-1242?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 324a690e83319de04c7d02d5412b8b332c874575
</p>

File: src/app/Scenes/Search/SearchArtworksContainer.tests.tsx [hunk 1]
```diff
@@ -80,7 +80,6 @@ describe("SearchArtworks", () => {
           "destination_screen_owner_type": "artwork",
           "position": 0,
           "query": "keyword",
-          "signal_label": "Limited-Time Offer",
```
Comment (nickskalkin) [inline src/app/Scenes/Search/SearchArtworksContainer.tests.tsx:83]: **question**: Is this an intentional change?

**Review by nickskalkin — APPROVED** [review]
✨

**Review by anandaroop — APPROVED** [review]
Nice! 

Looks good, but will probably need a schema bump after the code review changes in https://github.com/artsy/metaphysics/pull/5962

**Pull request merged**